tests/test_integrated_channels/test_blackboard/test_transmitters/test_learner_data.py

```diff
@@ -65,7 +65,7 @@ def test_transmit_success(self):
 
         transmitter = learner_data.BlackboardLearnerTransmitter(self.enterprise_config)
 
-        transmitter.transmit(self.exporter([self.payload]))
```

Comment:
Why only the blackboard tests? Do the Canvas tests not also need this change?

Reply:
ahh good points!